src/openrct2/interface/Viewport.cpp

```diff
@@ -1665,25 +1665,40 @@ void get_map_coordinates_from_pos(
 void get_map_coordinates_from_pos_window(
     rct_window* window, int32_t screenX, int32_t screenY, int32_t flags, int16_t* x, int16_t* y, int32_t* interactionType,
     TileElement** tileElement, rct_viewport** viewport)
+{
+  ScreenCoordsXY screenCoords(screenX, screenY);
+  CoordsXY mapCoords(0, 0);
```

Comment:
`CoordsXY` are already initialized to {0, 0} on their default constructor, so this seems superfluous

Reply:
Good point, changed to use the default constructor.